docs/tools/firmware.sh

```diff
@@ -515,103 +786,97 @@ case "$os" in
 		;;
 
 	(Linux)
-		echo "Detected Linux"
+		if [[ $verbose = "-v" ]]; then
+			echo "Detected Linux"
+		fi
 		if [[ ! -e /lib/firmware/brcm ]]; then
-			echo "/lib/firmware/brcm does not seem to exist. This script requires that directory to function."
-			echo "If you are on some exotic distro like NixOS, please check the wiki for more information:"
-			echo "  https://wiki.t2linux.org"
-			echo "Exiting..."
+			cat <<- EOF
+			/lib/firmware/brcm does not seem to exist. This script requires that directory to function.
+			If you are on some exotic distro like NixOS, please check the wiki for more information:
+			  https://wiki.t2linux.org
+			Exiting...
+			EOF
 			exit 1
 		fi
-		echo -e "\nHow do you want to copy the firmware to Linux?"
-		echo -e "\n1. Retrieve the firmware from the EFI partition."
-		echo "2. Retrieve the firmware directly from macOS."
-		echo "3. Download a macOS Recovery Image from Apple and extract the firmware from there."
-		echo -e "\nNote: If you are choosing Option 1, then make sure you have run the same script on macOS before and chose Option 1 (Copy the firmware to the EFI partition and run the same script on Linux to retrieve it) there."
-		read choice
+		cat <<- EOF
+		How do you want to copy the firmware to Linux?
+
+		1. Retrieve the firmware from the EFI partition.
+		2. Retrieve the firmware directly from macOS.
+		3. Download a macOS Recovery Image from Apple and extract the firmware from there.
+		
+		Note: If you are choosing Option 1, then make sure you have run the same script on macOS before and chose Option 1 (Copy the firmware to the EFI partition and run the same script on Linux to retrieve it) there.
+		EOF
+		read -r choice
 		case ${choice} in
 			(1)
-				echo -e "\nRe-mounting the EFI partition"
+				echo -e "Re-mounting the EFI partition"
```

Comment:
Why has a blank line been removed?

Reply:
I wasnt sure if the extra newlines were intentional, I think all of the newline changes are reverted now.